### PR TITLE
Add request parameter to get_unauthorised_redirect_url

### PR DESCRIPTION
**Pull request by adam-incuna**
Something that was missed previously.

@incuna/backend quick merge please?

### CODE REVIEW

**Comment by KevinEtienne** [comment]
It's strange I've have the `this pull request can be merged` but tests are still running on travis.

**Comment by adam-incuna** [comment]
@KevinEtienne That might be the `ci skip` confusing it :P

**Comment by KevinEtienne** [comment]
Tests just finished for the first commit but two didn't starts as a postgres command failed, I've restarted them.

**Comment by adam-incuna** [comment]
Ah, thanks. :)

**Comment by adam-incuna** [comment]
@KevinEtienne it is done at last :P

**Comment by adam-incuna** [comment]
thanks! :)

**Comment by KevinEtienne** [comment]
hurray!

File: changelog.rst [hunk 1]
```diff
@@ -1,6 +1,10 @@
 Changelog
 =========
 
+Upcoming
+--------
+* Enhancement: Add a request parameter to get_unauthorised_redirect_url.
```
Comment (meshy) [inline changelog.rst]: Don't forget that this is backwards incompatible! `;)`

**Pull request merged**